xAPI.md

```diff
@@ -3226,10 +3226,16 @@ include attachment raw data and MUST report application/json.
 
 ###### Filter Conditions for StatementRefs 
 
-For filter parameters which are not time or sequence based (that is, other than
-since, until, or limit), Statements which target another Statement (by using a StatementRef
+This section outlines rules by which additional Statements are considered to meet the filter coniditions even
+if they do not match the original query's filter parameters but instead target a Statement which does. 
+These rules **do not** apply when retrieving a single Statement with statementId or voidedStatementId.
+
+For filter parameters which are not time or sequence based (that is, other than since, until, or limit), 
+Statements which target another Statement (by using a StatementRef
 as the Object of the Statement) will meet the filter condition if the targeted Statement meets 
-the condition. The time and sequence based parameters MUST still be applied to the Statement 
+the condition of that filter parameter.
```

Comment:
in the previous text we used "meet the filter condition/meets the condition" intentionally, as the rule applies to itself. This changes it to "meet the filter condition/meets the condition of that filter parameter". I'm not actually sure what the second bit means, but this makes it hard to read recursively.

Actually, I think that by changing it to talk about singular filter parameters, this text actually breaks the intended meaning. The previous wording is dense and a bit hard to follow, but exact, and I don't want to replace it with anything that isn't equally exact (as this is a pretty important part of the spec).